services/movie.py

```diff
@@ -0,0 +1,34 @@
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: None = None,
+        actors_ids: None = None

```

Comment:
The default values for `genres_ids` and `actors_ids` should be `None`, but the type hint should be `Optional[List[int]]` to indicate that these parameters can be lists of integers or `None`.